d3a_interface/validators/__init__.py

```diff
@@ -0,0 +1,35 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of D3A.
+
+This program is free software: you can redistribute it and/or modify it under the terms of the
+GNU General Public License as published by the Free Software Foundation, either version 3 of the
+License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See
+the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with this program. If
+not, see <http://www.gnu.org/licenses/>.
+"""
+from d3a_interface.validators.cep_validator import CommercialProducerValidator
+from d3a_interface.validators.finite_diesel_generator_validator import (
+    FiniteDieselGeneratorValidator)
+from d3a_interface.validators.home_meter_validator import HomeMeterValidator
+from d3a_interface.validators.infinite_bus_validator import InfiniteBusValidator
+from d3a_interface.validators.load_validator import LoadValidator
+from d3a_interface.validators.market_maker_validator import MarketMakerValidator
+from d3a_interface.validators.pv_validator import PVValidator
+from d3a_interface.validators.storage_validator import StorageValidator
+
+# Explicitly declare the names of the module's public API
```

Comment:
Sorry if this has been discussed in one of the other PRs. Can you maybe point me to the discussion and explain why this is needed?

Reply:
btw pep states that the dunder names should be placed before imports https://www.python.org/dev/peps/pep-0008/#id24

Reply:
@hannesdiedrich Adding the imports in `__init__.py` allows to import the validators directly from the `validators` package, without the specific module:
```python
from d3a_interface.validators import CommercialProducerValidator
# Instead of this
from d3a_interface.validators.cep_validator import CommercialProducerValidator
```
However, flake8 fails if we just write the imports:
> `./d3a_interface/validators/__init__.py:17:1`: F401 'd3a_interface.validators.cep_validator.CommercialProducerValidator' imported but unused

One option would be to ignore this using `#noqa` or similar. Another approach is to use `__all__`. From PEP8:
> To better support introspection, modules should explicitly declare the names in their public API using the `__all__` attribute. Setting `__all__` to an empty list indicates that the module has no public API.

This is especially useful to limit what gets imported when you write `from d3a_interface.validators import *` (see [Importing from a package](https://docs.python.org/3.8/tutorial/modules.html#importing-from-a-package)).

@Hassan754 interesting, I didn't know that! Thanks, I'm moving the imports below then.

Reply:
Thanks a lot! Makes sense and I will consider this way for my next implementations.